src/WebJobs.Script/BindingExtensions/ExtensionsManager.cs

```diff
@@ -202,6 +202,8 @@ internal virtual Task ProcessExtensionsProject(string projectFolder)
         private void SetupProcessEnvironment(ProcessStartInfo startInfo)
         {
             TryAdd(startInfo.Environment, EnvironmentSettingNames.DotnetSkipFirstTimeExperience, "true");
+            TryAdd(startInfo.Environment, EnvironmentSettingNames.DotnetAddGlobalToolsToPath, "false");
+            TryAdd(startInfo.Environment, EnvironmentSettingNames.DotnetNoLogo, "true");
```

Comment:
Do you have an example of what gets printed now? I hate that big block of text... but seeing the .NET SDK version written out may be helpful if we need to debug anything. Does that still show up anywhere?

Reply:
It looks like this with "NOLOGO"

```
2020-08-19T21:15:49.910 [Information] Starting packages restore
2020-08-19T21:15:55.008 [Information] Determining projects to restore...
2020-08-19T21:15:57.433 [Information] Restored D:\local\Temp\3a0c7ec1-93d5-462e-a796-213d217cb1e4\function.proj (in 1.6 sec).
2020-08-19T21:15:57.816 [Information] Packages restored.
2020-08-19T21:15:57.816 [Information] Package references have changed.
```

This is also the current experience, and the logo / welcome message would be new (and show up in app insights)